yusukenishikado/README.md

```diff
@@ -0,0 +1,21 @@
+# About me
+
+This is Yusuke Nishikado.
+I am full stack developer and I have rich experience with frontend and backend development.
+
+## Major skill
+
+- HTML
+- CSS
+- JavaScript
+- PHP
+- Laravel
+- Mern Stack
+
+## Email Address
+
+My email address is [yusukenishikado96@gmail.com]
+
+## Applications
+
+- [todo](./todo/README.md)
```

Comment:
don't point README.md, but the whole todo directory

Reply:
I resolved problem